Sources/Mantis/RotationDial/DialConfig.swift

```diff
@@ -54,12 +54,7 @@ enum DialConfig {
         case useDefault
         case custom(CGPoint)
     }
-    
-    enum RotationLimitType {
-        case noLimit
-        case limit(angle: CGAngle)
-    }
-    
+
     enum AnglehowLimitType {
```

Comment:
AnglehowLimitType is a typo I made before which should be `AngleShowLimitType`. It would be great if you can correct it for me.